sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2758 @@
+!! @param [out] d(2) - o number density(cm-3)                                  
+!! @param [out] d(3) - n2 number density(cm-3)                                 
+!! @param [out] d(4) - o2 number density(cm-3)                                 
+!! @param [out] d(5) - ar number density(cm-3)                                 
+!! @param [out] d(6) - total mass density(gm/cm3)                              
+!! @param [out] d(7) - h number density(cm-3)                                  
+!! @param [out] d(8) - n number density(cm-3)                                  
+!! @param [out] d(9) - anomalous oxygen number density(cm-3)                   
+!! @param [out] t(1) - exospheric temperature                                  
+!! @param [out] t(2) - temperature at alt                                      
+!!                                                                      
+!!    Notes on output variables:                                        
+!!       To get output in m-3 and kg/m3:   call meters(.true.)          
+!!                                                                      
+!!       o, h, and n are set to zero below 72.5 km                      
+!!                                                                      
+!!       t(1), exospheric temperature, is set to global average for     
+!!       altitudes below 120 km. the 120 km gradient is left at global  
+!!       average value for altitudes below 72 km.                       
+!!                                                                      
+!!       d(6), total mass density, is not the same for subroutines gtd7 
+!!       and gtd7d                                                      
+!!                                                                      
+!!         subroutine gtd7 -- d(6) is the sum of the mass densities of t
+!!         species labeled by indices 1-5 and 7-8 in output variable d. 
+!!         this includes he, o, n2, o2, ar, h, and n but does not includ
+!!         anomalous oxygen (species index 9).                          
+!!                                                                      
+!!         subroutine gtd7d -- d(6) is the "effective total mass density
+!!         for drag" and is the sum of the mass densities of all species
+!!         in this model, including anomalous oxygen.                   
+!!                                                                      
+!!    switches: the following is for test and special purposes:         
+!!                                                                      
+!!       To turn on and off particular variations call tselec(sw),      
+!!       where sw is a 25 element array containing 0. for off, 1.       
+!!       for on, or 2. for main effects off but cross terms on          
+!!       for the following variations                                   
+!!              1 - f10.7 effect on mean  2 - time independent          
```

Comment:
I think this will look poorly in doxygen. Put each number on its own line to get a numbered list:

1. Item 1
2. Item 2
3. Item 3
...

Please do not add all the extra spaces, it makes the documentation hard to maintain and harder to get looking good in doxygen. Instead of using indentation and extra spaces, use doxygen formatting elements like lists, tables, and section headings.